src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -44,24 +44,19 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
     }
   }
 
-  override def saveKeyValue(userInfo: UserInfo, key: String, value: String): Future[Try[Unit]] = {
+  override def saveKeyValues(userInfo: UserInfo, keyValues: Map[String, String]): Future[Try[Unit]] = {
```

Comment:
A set of keys with associated values already has a name: a map. So, perhaps, instead of `keyValues`, `map`? Unless there is some value in inventing our own term for this.